armory/docker/images.py

```diff
@@ -15,22 +15,18 @@
 
 DOCKER_REPOSITORY = "twosixarmory"
 
-PYTORCH = f"{DOCKER_REPOSITORY}/pytorch:{TAG}"
+ARMORY_IMAGE_NAME = f"{DOCKER_REPOSITORY}/armory:{TAG}"
 PYTORCH_DEEPSPEECH = f"{DOCKER_REPOSITORY}/pytorch-deepspeech:{TAG}"
-TF2 = f"{DOCKER_REPOSITORY}/tf2:{TAG}"
-CARLA_MOT = f"{DOCKER_REPOSITORY}/carla-mot:{TAG}"
 ALL = (
-    PYTORCH,
+    ARMORY_IMAGE_NAME,
     PYTORCH_DEEPSPEECH,
-    TF2,
-    CARLA_MOT,
 )
 REPOSITORIES = tuple(x.split(":")[0] for x in ALL)
 IMAGE_MAP = {
-    "pytorch": PYTORCH,
+    "pytorch": ARMORY_IMAGE_NAME,
```

Comment:
Can you add a key-value pair `"armory": ARMORY_IMAGE_NAME,`?

This dict is used in `__main__` and in image lookups, and this would enable using `armory` on the command line or the scenario configs.